mopidy/http/handlers.py

```diff
@@ -20,17 +20,23 @@
 
 def make_mopidy_app_factory(apps, statics):
     def mopidy_app_factory(config, core):
+        disable_csrf_protection = not config['http']['csrf_protection']
```

Comment:
You need to rename this variable and flip the logic here.

Reply:
Ok, I actually misread this and didn't notice the "not". I guess that proves my point. :-)